themes/mcf/components/AboutTheFunds.tsx

```diff
@@ -11,19 +11,20 @@ type FundDescription = {
 
 const fundDescriptions: FundDescription[] = [
   {
-    name: "Adaptation Fund",
+    name: "Adaptation Fund ",
     description: "(AF) finances projects and programmes that help vulnerable communities in developing countries adapt to climate change.",
   },
   {
-    name: "Climate Investment Funds",
+    name: "Climate Investment Funds ",
     description: "(CIF) is a major multilateral climate fund established in 2008 to finance climate-smart solutions in developing countries.",
   },
   {
-    name: "Global Environment Facility",
-    description: "(GEF) projects and programmes that help vulnerable communities in developing countries adapt to climate change.",
+    name: "The Global Environment Facility ",
+    description:
+      "(GEF) is a multilateral family of funds dedicated to confronting biodiversity loss, climate change, and pollution, and supporting land and ocean health.",
   },
   {
-    name: "Green Climate Fund",
+    name: "Green Climate Fund ",
```

Comment:
Trailing whitespace -- perhaps it's better to put the whitespace after wherever we're referencing these name values so we keep the names themselves clean😉

Reply:
@katybaulch  do you think it looks better here, after `fund.name`

```            
<p key={fund.name}>
              <strong>{fund.name} </strong>
              {fund.description}
              <br />
              <br />
            </p>
```